test/format/util.c

```diff
@@ -253,10 +253,22 @@ timestamp_once(WT_SESSION *session)
     if (LOCK_INITIALIZED(&g.ts_lock))
         lock_writelock(session, &g.ts_lock);
 
-    ret = conn->query_timestamp(conn, buf + strlen(oldest_timestamp_str), "get=all_durable");
+    ret = conn->query_timestamp(conn, tsbuf, "get=all_durable");
     testutil_assert(ret == 0 || ret == WT_NOTFOUND);
-    if (ret == 0)
+    if (ret == 0) {
+        testutil_check(__wt_snprintf(buf, sizeof(buf), "%s%s", oldest_timestamp_str, tsbuf));
+        /*
+         * When we're doing rollback to stable operations, we'll advance the stable timestamp too,
+         * in lockstep with the oldest timestamp.
+         */
+        if (g.c_txn_rollback_to_stable) {
+            len = strlen(buf);
+            WT_ASSERT((WT_SESSION_IMPL *)session, len < sizeof(buf));
+            testutil_check(
+              __wt_snprintf(buf + len, sizeof(buf) - len, ",%s%s", stable_timestamp_str, tsbuf));
+        }
```

Comment:
Why do we move the stable timestamp in lockstep with the oldest timestamp, wouldn't they typically be different values in real applications?